spec-main/events-helpers.ts

```diff
@@ -0,0 +1,40 @@
+import { EventEmitter } from "electron";
```

Comment:
i don't love that this is copy/pasted in two places. it'll get out of sync quick. let's find a way to deduplicate this.

Reply:
I de-duped it 🎉

Reply:
This doesn't seem to be deduped?

Reply:
As discussed not possible rn